content/howtotech.md

```diff
@@ -0,0 +1,59 @@
+---
+title: First steps - How to hack
+---
+
+
+To make the most of your time analyzing data and minimize technical hiccups along the way, it's essential to start off on the right foot. This guide lays out the key first steps and to-dos to help you set up everything properly and get started quickly.
+
+In this documentation, we’ll walk you through the basic setup and essential technical preparations. For more detailed explanations or deeper dives, we’ll point you to relevant additional resources. Take a moment to read through everything carefully—it’ll save you time and effort later on!
+
+---
+
+## 1. Check your access to the [HPC system Levante](https://www.dkrz.de/en/systems/hpc/hlre-4-levante?set_language=en)
+
+To get access to the data you will need to get an [**DKRZ user account**](https://docs.dkrz.de/doc/getting_started/getting-a-user-account/dkrz-user-account.html#dkrz-user-account).
+You need to [register](https://docs.dkrz.de/doc/getting_started/getting-a-user-account/dkrz-user-account.html#creating-a-new-account-from-scratch) here. Mention, that you are part of the global hackathon and [join an existing project](https://docs.dkrz.de/doc/getting_started/getting-a-user-account/dkrz-user-account.html#join-existing-project). The ID for the project is *bb1153*.
+
+
+## 2. Log in
+
+Once you are registered, use either https://jupyterhub.dkrz.de or `ssh -Y <userid>@levante.dkrz.de` for *shell logins*.
+
+Levante uses the queuing system [SLURM](https://docs.dkrz.de/doc/levante/running-jobs/index.html) for submission, scheduling, execution, and monitoring of jobs. Choose the **interactive** partition for interactive data analysis.
+
+Information about the file system or **where to store what on Levante**
+* /home/ for scripts. Backuped, max 30 GB/user
+* /work/ for data you need more than 2 weeks (model output/…), project quota, NO BACKUP
+* /scratch/ for temporary files, 15 TB quota - DELETION AFTER TWO WEEKS, NO BACKUP
+
+
+## 3. Set up your python environment
+
+[*Input needed...*]
```

Comment:
I would either tell users how to register their own Jupyter kernel, or point the the official hackathon kernel. Maybe @florianziemen knows if DKRZ will add it to the public list